index.js

```diff
@@ -10,11 +11,15 @@ const fieldOfActivityRouter = require('./app/routes/fieldOfActivity');
 const companyRouter = require('./app/routes/company');
 const sequelizeErrorParser = require('./app/services/sequelizeErrorParser');
 
+global.__basedir = __dirname;
```

Comment:
Please avoid using global variables. A better approach is to use a config file where you define the tmp folder and the one used as final destination. Then you can import that config everywhere.